backend/typescript/middlewares/validators/checkInValidators.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from "express";
+import { getApiValidationError, validatePrimitive } from "./util";
+
+const checkInDtoValidator = async (
```

Comment:
planning on using linna's and dora's validators to check if start date is greater than end date

Reply:
but isnt that for body contents not query params?

Reply:
oh yeah that's true my bad - i dont think we can validate it through middlewares then (?) i think jessie has a validation in her function